components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -174,6 +183,30 @@ public static void createApplication(String appName, String appOwner, String app
         localAndOutboundAuthenticationConfig.setUseTenantDomainInLocalSubjectIdentifier(true);
         localAndOutboundAuthenticationConfig.setSkipConsent(true);
         localAndOutboundAuthenticationConfig.setSkipLogoutConsent(true);
+
+        if (CONSOLE_APP.equals(appName)) {
+            AuthenticationStep authenticationStep1 = new AuthenticationStep();
+            LocalAuthenticatorConfig identifierFirst = new LocalAuthenticatorConfig();
+            identifierFirst.setName("IdentifierExecutor");
+            identifierFirst.setDisplayName("identifier-first");
+            authenticationStep1.setLocalAuthenticatorConfigs(new LocalAuthenticatorConfig[]{identifierFirst});
+            authenticationStep1.setSubjectStep(false);
+            authenticationStep1.setAttributeStep(false);
+            authenticationStep1.setStepOrder(1);
+
+            AuthenticationStep authenticationStep2 = new AuthenticationStep();
+            LocalAuthenticatorConfig basic = new LocalAuthenticatorConfig();
+            basic.setName("BasicAuthenticator");
+            basic.setDisplayName("basic");
```

Comment:
Shall we use constants if they are available?

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/3556